### PR TITLE
Fix @param null on RequestTrait

### PR DESCRIPTION
**Pull request by samsonasik**
**Description**

Same with PR:

- https://github.com/codeigniter4/CodeIgniter4/pull/7322

This PR fix `@param null` on RequestTrait.

**Checklist:**
- [x] Securely signed commits

### CODE REVIEW

File: system/HTTP/RequestTrait.php [hunk 1]
```diff
@@ -191,8 +191,8 @@ private function getClientIP(string $header): ?string
      * Fetch an item from the $_SERVER array.
      *
      * @param array|string|null $index  Index for item to be fetched from $_SERVER
-     * @param int|null          $filter A filter name to be applied
-     * @param null              $flags
+     * @param int|string|null   $filter A filter name to be applied
```
Comment (kenjis) [inline system/HTTP/RequestTrait.php:194]: Remove `string`.

File: system/HTTP/RequestTrait.php [hunk 2]
```diff
@@ -204,9 +204,9 @@ public function getServer($index = null, $filter = null, $flags = null)
     /**
      * Fetch an item from the $_ENV array.
      *
-     * @param null $index  Index for item to be fetched from $_ENV
-     * @param null $filter A filter name to be applied
-     * @param null $flags
+     * @param string|null     $index  Index for item to be fetched from $_ENV
+     * @param int|string|null $filter A filter name to be applied
```
Comment (kenjis) [inline system/HTTP/RequestTrait.php:208]: Remove `string`.

File: system/HTTP/RequestTrait.php [hunk 3]
```diff
@@ -204,9 +204,9 @@ public function getServer($index = null, $filter = null, $flags = null)
     /**
      * Fetch an item from the $_ENV array.
      *
-     * @param null $index  Index for item to be fetched from $_ENV
-     * @param null $filter A filter name to be applied
-     * @param null $flags
+     * @param string|null     $index  Index for item to be fetched from $_ENV
```
Comment (kenjis) [inline system/HTTP/RequestTrait.php:207]: `array|string|null`
See `fetchGlobal()`.

File: system/HTTP/RequestTrait.php (hunk 1 above)
Comment (samsonasik) [reply system/HTTP/RequestTrait.php]: Ok, removed https://github.com/codeigniter4/CodeIgniter4/pull/7323/commits/0ead60be3ec798a671e5d63d770054119db35f48

File: system/HTTP/RequestTrait.php (hunk 2 above)
Comment (samsonasik) [reply system/HTTP/RequestTrait.php]: Ok, removed https://github.com/codeigniter4/CodeIgniter4/pull/7323/commits/0ead60be3ec798a671e5d63d770054119db35f48

File: system/HTTP/RequestTrait.php (hunk 3 above)
Comment (samsonasik) [reply system/HTTP/RequestTrait.php]: Ok, removed https://github.com/codeigniter4/CodeIgniter4/pull/7323/commits/0ead60be3ec798a671e5d63d770054119db35f48

**Review by kenjis — APPROVED** [review]
Thank you!

**Review by paulbalandan — APPROVED** [review]

**Comment by samsonasik** [comment]
@kenjis @paulbalandan thank you for the review.

**Pull request merged**